doc/deploy.md

```diff
@@ -0,0 +1,82 @@
+# Deploy
+
+https://kind.sigs.k8s.io/docs/user/ingress#ingress-nginx
+
+## Create cluster
+```bash
+kind create cluster --name tinyhen-test --config deploy/cluster-config.yaml
+```
+
+Expected output:
+```bash
+Creating cluster "tinyhen-test" ...
+ ✓ Ensuring node image (kindest/node:v1.24.0) 🖼 
+ ✓ Preparing nodes 📦  
```

Comment:
`ERROR: failed to create cluster: command "docker run --name tinyhen-control-plane --hostname tinyhen-control-plane --label io.x-k8s.kind.role=control-plane --privileged --security-opt seccomp=unconfined --security-opt apparmor=unconfined --tmpfs /tmp --tmpfs /run --volume /var --volume /lib/modules:/lib/modules:ro -e KIND_EXPERIMENTAL_CONTAINERD_SNAPSHOTTER --detach --tty --label io.x-k8s.kind.cluster=tinyhen --net kind --restart=on-failure:1 --init=false --publish=0.0.0.0:80:80/TCP --publish=0.0.0.0:443:443/TCP --publish=127.0.0.1:42247:6443/TCP -e KUBECONFIG=/etc/kubernetes/admin.conf kindest/node:v1.24.0@sha256:406fd86d48eaf4c04c7280cd1d2ca1d61e7d0d61ddef0125cb097bc7b82ed6a1" failed with error: exit status 125
Command Output: 473b259f899ff1a5e812dcbfeb777dfba22cda1a8365782d905407e6d6f10dd9
docker: Error response from daemon: driver failed programming external connectivity on endpoint tinyhen-control-plane (cc891218944e96f4a76863a32382831877740bc140bac4c5172e3049d80d594f): Error starting userland proxy: listen tcp4 0.0.0.0:80: bind: address already in use.`

(and 
```
github.com/doniacld/tiny-hen  dev ✔
▶ lsof -i :80

github.com/doniacld/tiny-hen  dev ✔
▶
```
)

Reply:
🤔